benchmark/camelyon/benchmarks.py

```diff
@@ -64,7 +64,7 @@ def fed_avg(params: dict, train_folder: Path, test_folder: Path):
         mode=exp_params["mode"],
         cp_name=exp_params["cp_name"],
         cancel_cp=exp_params["cancel_cp"],
-        torch_gpu=exp_params["torch_gpu"],
```

Comment:
This will break the CI, need to do a companion PR on `test-rekease-dev.yaml`

Reply:
Yes, this is identified, but thanks for the reminder!

Reply:
https://github.com/owkin/substra-ci/pull/174/commits/dbc718caaf32a04e38700eee48f7ca35bef2ad4d